src/Umbraco.Web.UI.Client/src/views/content/overlays/publishdescendants.html

```diff
@@ -6,12 +6,15 @@
         </div>
 
         <div class="flex mb3">
-            <umb-toggle
-                checked="vm.includeUnpublished"
-                on-click="vm.toggleIncludeUnpublished()"
-                class="mr2"
-            />
-            <localize key="content_includeUnpublished"></localize>
+
+            <umb-toggle checked="vm.includeUnpublished"
+                        on-click="vm.toggleIncludeUnpublished()"
+                        class="mr2"
+                        label-on="{{vm.labels.includeUnpublished}}"
+                        label-off="{{vm.labels.includeUnpublished}}"
+                        label-position="right"
+                        show-labels="true"/>
+
```

Comment:
@clausjensen notice angular directives/components can't be void elements (self-closing), so it needs a end-closing `</umb-toggle>`.

There is a great explanation of it here: https://stackoverflow.com/a/25012451/1693918

Also here is an example of what can happen when a component/directive is self-closed. Of course native HTML5 elements can still be self-closed like `<img />`, `<br />` etc.
https://github.com/umbraco/Umbraco-CMS/pull/8478

Here is a PR updating this for `<umb-checkbox>` and `<umb-radiobutton>`, but should also be done for `<umb-toggle>`.
https://github.com/umbraco/Umbraco-CMS/pull/8547

Reply:
I've merged your PR as it might be good just for history .. and the whitespace change is fine. Merging this afterwards will overwrite some things in yours which is fine.